frontend/src/components/IssueModeSelector/IssueModeSelector.module.css

```diff
@@ -0,0 +1,24 @@
+.tagSwitch {
+	display: flex;
+	height: 24px;
+	padding: 2px 6px;
+	justify-content: center;
+	align-items: center;
+	gap: 2px;
+	flex: 1 0 0;
+	border-radius: 6px;
+	cursor: pointer;
+}
+
+.activeSwitch {
+	background-color: var(--color-purple);
+	background: var(--color-purple);
+	color: var(--color-white);
+	box-shadow: 0px -1px 0px 0px rgba(0, 0, 0, 0.32) inset
+}
+
+.inactiveSwitch {
+	borderColor: var(--color-gray-300);
+	color: var(--_11tb6c5c);
```

Comment:
Where did this variable come from? I'm wondering if this is a hash produced during a build, which would be unreliable to reference long term. Is there another variable we could reference? Or else maybe it would be better to use the `color` prop on the `Text` component here.

```tsx
<Text color={selected ? 'secondaryContentText' : 'white'}>...</Text>
```

Reply:
I thought it was a global css variable but the naming makes me believe as you do that it was generated by a build script whose name may change in the future - so cannot be relied upon.

Using the color prop on the Text component is the more “idiomatic” way. Will do that instead.